docs/kernel-policy.md

```diff
@@ -84,6 +85,24 @@ the minimal configuration for a guest kernel for a successful microVM boot is:
     - `CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=y`
     - `CONFIG_KVM_GUEST=y`.
 
+##### Booting with ACPI (x86_64 only):
+
+Firecracker supports booting kernels with ACPI support. The relevant
+configurations for the guest kernel are:
+
+- `CONFIG_ACPI=y`
+- `CONFIG_PCI=y`
+
+Please note that Firecracker does not support PCI devices. The `CONFIG_PCI`
+option is needed for ACPI initialization inside the guest.
+
+If users want to disable the old way of booting (via MPTable and passing VirtIO
+devices as kernel command line parameters) they should disable the following two
+options:
+
+- `CONFIG_X86_MPPARSE`
+- `CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES`
+
```

Comment:
We should also mention the 2nd device initialisation failures which is ignored by kernel when ACPI is enabled and Firecracker passes mptable+acpi tables

Reply:
Added some text regarding what exactly is supported and mention the warning messages.